File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.py
def solution(A, B):
    answer = 0
    A.sort()
    B.sort()
    while len(B) > 0:
        if B[0] <= A[0]:
            B.pop(0)
        else:
            answer += 1
            A.pop(0)
            B.pop(0)
    return answer

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.java
import java.util.*;

class Solution {
    public int solution(int[] A, int[] B) {
        int answer = 0;
        
        List<Integer> a = new ArrayList<>();
        List<Integer> b = new ArrayList<>();
        for (int i = 0; i < A.length; i++) {
            a.add(A[i]);
            b.add(B[i]);
        }
        Collections.sort(a);
        Collections.sort(b);
        
        while(b.size() > 0) {
            if (a.get(0) >= b.get(0)) {
                b.remove(0);
            } else {
                answer += 1;
                a.remove(0);
                b.remove(0);
            }
        }
        
        return answer;
    }
}